roles/pulibrary.oar/templates/local_settings.cfg.j2

```diff
@@ -0,0 +1,34 @@
+# {{ ansible_managed | comment }}
+DSPACE_BASE="/home/{{ deploy_user }}"
+DSPACE_DIR="$DSPACE_BASE/dspace"
+DSPACE_BUILD_CFG={{ env }}
+
+ANT_HOME=/opt/apache-ant
+ANT_OPTS="-Xms128m -Xmx800m"
+DSPACE_CLEAN_BACKUPS_ON_DEPLOY=false
+DSPACE_DB={{ dspace_db_name }}
+DSPACE_DB_BACKUP="$DSPACE_DIR/backup"
+DSPACE_DB_FILENAME=dspace
+DSPACE_DB_USER={{ dspace_db_user }}
+DSPACE_MODE=TEST
+DSPACE_OVERWRITE_ON_DEPLOY=true
+DSPACE_REMOVE_LOGS_AT_STARTUP=false
+# DSPACE_REMOVE_WEBAPPS="xmlui oai rest rdf sword swordv2"
```

Comment:
By default this will ensure that the `oai` and `rest` modules are *not* built - should this be the case? Atmire has also strongly suggested that we explore attempting to use `sword` in the future for better Thesis Central (Vireo) support (although another PR for this is best).